src/foundry/client/pixi/groups/primary.d.mts

```diff
@@ -93,17 +93,17 @@ declare global {
     /**
      * The collection of PrimaryDrawingContainer objects which are rendered in the Scene.
      */
-    drawings: Collection<DrawingShape>;
+    drawings: Collection<PrimaryGraphics>;
```

Comment:
These are based upon usage as some of these are documented incorrectly. Filed this to Foundry.